benchmarks/src/commonMain/kotlin/BufferOps.kt

```diff
@@ -393,6 +393,7 @@ open class IndexOfByteString {
 
     private var buffer = Buffer()
     private var byteString = ByteString()
+    private var source: Source = buffer
```

Comment:
Is this field only needed for extension function static selection?

Reply:
Yes, I committed that change accidentally (added it locally to ensure that Source::indexOf is also faster), but then thought that it's worth leaving it.
I think that the better solution is to introduce yet another benchmark.

Reply:
Filed https://github.com/Kotlin/kotlinx-io/issues/243